lib/deas/test_helpers.rb

```diff
@@ -1,12 +1,16 @@
+require 'rack/request'
+require 'rack/response'
 require 'deas/test_runner'
 
 module Deas
 
   module TestHelpers
 
-    module_function
```

Comment:
@kellyredding - Any specific reason for removing this? If we want to do this we should probably be sure to update Sanford as well so they behave consistently.